File: datasets/IR-Plag/convertai-original-java-rest-typescript/case-05/plagiarized/L1/05/method.ts
import * as readline from 'readline';

class Method {
  // Main program
  public static main(args: string[]): void {
    const rl = readline.createInterface({
      input: process.stdin,
      output: process.stdout
    });

    rl.question('Enter an integer: ', (numberStr) => {
      const number = parseInt(numberStr);
      // Call the reverse method
      this.reverse(number);
      rl.close();
    });
  }

  // Reverse method
  public static reverse(number: number): void {
    let reversedNumber = '';
    while (number !== 0) {
      const remainder = number % 10;
      reversedNumber += remainder;
      number = Math.floor(number / 10);
    }
    console.log(reversedNumber);
  }
}

Method.main(process.argv.slice(2));

File: datasets/IR-Plag/convertai-original-java-rest-python/case-05/plagiarized/L1/06/method.py
import sys

def reverse(number):
    while number != 0:
        remainder = number % 10
        print(remainder, end='')
        number = number // 10
    print()

if __name__ == "__main__":
    print("Enter an integer: ", end='')
    number = int(input())
    reverse(number)

